requirements.txt

```diff
@@ -1,2 +1,3 @@
 messytables==0.15.2
 Unidecode==1.0.22
+#six  # Is inside ckan, required to be python 2/3 compatible
```

Comment:
Would it make sense to add `six` as a dependency, without specifying a version, just as a sanity-check? We don't expect to need to install it, but if for some reason it's *not* installed, then fail fast.

Reply:
from most of the work i've reviewed in improving the cicd pipelines, there is two camps. one is to match whats in ckan, the other is to leave it blank but then it will go to latest version, or just leave it along as extensions are installed post ckan so it should not have any modules that ckan automatically installs just in case you upgrade ckan and then get it downgraded.

* locking to a version has issues in that ckan may go up to the next version and then gets downgraded (or vice versa)
* having it stated but not locked may install latest version and give issues or if requirements are -u (upgraded) it will take latest which has issues
* leaving it out with readme stating ckan core must be installed prior is best bet so not to interfere with ckan.